src/main/java/ecdar/presentations/ModelPresentation.java

```diff
@@ -87,7 +85,7 @@ private void initializeName() {
         updateColor.run();
 
         // Center the text vertically and aff a left padding of CORNER_SIZE
-        controller.name.setPadding(new Insets(2, 0, 0, Grid.CORNER_SIZE));
+        controller.name.setPadding(new Insets(2, 0, 0, 40));
```

Comment:
Comment is outdated and consider a function of `Ecdar.CANVAS_PADDING`

Reply:
`PADDING` constant added